### PR TITLE
New test and solution for #10

### PR DESCRIPTION
**Pull request by pkdotson**
This problem sucked!

### CODE REVIEW

File: solutions/10.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      console.log('i went thru', num);
+      if (num%i === 0){
+        return false;
+      }
+    }
+    return true;
+  };
+
+for (num=0; num<arrPrime.length; num++){
+  console.log('i start', num);
```
Comment (msach22) [inline solutions/10.js]: remove console.log

File: solutions/10.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      if (primeConfirm(arrPrime[num]) === true || arrPrime[num] === 2){
+            console.log('ADDED', arrPrime[num]);
```
Comment (msach22) [inline solutions/10.js]: remove console.log

File: test/10.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const expect = require('chai').expect;
+const SumSolution = require('../solutions/10');
+const sumOfPrimeNums = SumSolution.sumOfPrimeNums;
+
+describe('Prime num total', () => {
+  it.only('should return sum of prime', () => {
+    const arrPrime = [1,2,3,9,8,7];
+    const expected = 12;
+    expect(sumOfPrimeNums(arrPrime)).to.equal(expected);
```
Comment (msach22) [inline test/10.js]: no need to create an expected variable since it's only being used once :)

File: solutions/10.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      if (num%i === 0){
+        return false;
+      }
+    }
+    return true;
+  };
+
+for (num=0; num<arrPrime.length; num++){
+  console.log('i start', num);
+      if (primeConfirm(arrPrime[num]) === true || arrPrime[num] === 2){
```
Comment (billma) [inline solutions/10.js]: In your primeConfirm fucntion, if you put  `if(num === 2) { return true }` the same way you did for checking if 1 is a prime number, you can remove `arrPrime[num] === 2` from the conditions

**Pull request closed without merging**